docs/examples/model_ex/model_all_comparisons.py

```diff
@@ -1,8 +1,8 @@
 """
-All data/model comparisons
-==========================
+Data/model comparisons
+======================
 
-This example shows every possible comparison between data and model.
+Multiple comparisons between data and model.
```

Comment:
Why omit the fact that it is indeed all the possible comparisons that make sense with `plot_data_model_comparison()`?